docs/guides/admin/docs/index.md

```diff
@@ -44,5 +44,5 @@ The Opencast Release Documentation is the official Opencast documentation for ea
 Further Documentation
 ---------------------
 
-In addition to this official documentation, further guides and tips can be found in the on the mailing lists, the IRC
-channel and at the regular meetings.
+In addition to this official documentation, further guides and tips can be found in the mailing lists, the IRC
+channel, and at the regular meetings.
```

Comment:
I guess, you won't really find guides in any of the mentioned places.
What about:

```suggestion
In addition to this official documentation, you can get help on the mailing lists, the IRC
channel, and at the regular meetings.
You can find all of these linked on [docs.opencast.org](https://docs.opencast.org).
```

Reply:
Looks good